compiler/compiler.py

```diff
@@ -317,8 +348,11 @@ def compile_source(source: SourcePackage,
         args.append(f'-U {tex_tree_timestamp}')
 
     logger.debug('run image %s with args %s', image, args)
-    run_docker(image, args=args,
-               volumes=[(host_source_dir, '/autotex')])
+    code, stdout, stderr = run_docker(image, args=args,
+                                      volumes=[(host_source_dir, '/autotex')])
+    # TODO: not sure about this.
```

Comment:
Not sure about this in a "exclude commented lines" way or in a "I want to include these lines but haven't tested" way?

I think that looks reasonable.

Reply:
I originally thought this was indicative of a problem with the tarball. Now I'm not so sure. Seems to be fine, but will continue to investigate.